packages/sanity/src/structure/comments/src/components/list/CommentsListItem.tsx

```diff
@@ -300,6 +317,8 @@ export const CommentsListItem = React.memo(function CommentsListItem(props: Comm
       >
         <Stack as="li" {...applyCommentIdAttr(parentComment._id)}>
           <CommentsListItemLayout
+            withAvatar={avatarConfig?.parentCommentAvatar}
```

Comment:
Nitpick: since all the other props are alphabetized, it would be nice to follow that pattern for the `withAvatar` prop as well 😇